### DISCUSSION TITLE
Where should I put the index.html file?

### DISCUSSION THREAD

**Discussion by MegWildrick** [chriskthomas/linkfree-generator#5] [Q&A] [answered]
The preview option gives me a view of the page that I want, but how do I make it public?

**Comment by chriskthomas** [comment, marked as answer]
You could upload it to GitHub Pages by adding it as the only file in a repository called `megwildrick.github.io` (or `user.github.io` in general). It would also work with Cloudflare Pages using the upload option or any other service like Netlify or Vercel.
